runtime/runtime.go

```diff
@@ -289,6 +289,14 @@ func (rt *Runtime) StartServer(ctx context.Context) {
 // will block until either: an error occurs, the context is canceled, or
 // a SIGTERM or SIGKILL signal is sent.
 func (rt *Runtime) Serve(ctx context.Context) error {
+
+	undo, err := maxprocs.Set()
```

Comment:
❓ We could pass a logger. It might be worthwhile to do that, right now, the effect of this might go unnoticed. Debug level, perhaps?

Reply:
Sending a logger into the library now.